src/dropdown/Dropdown.js

```diff
@@ -9,22 +10,16 @@ const themes = {
 };
 
 const PropTypes = {
-  children: React.PropTypes.oneOfType([
-    React.PropTypes.node,
-    React.PropTypes.element,
-    React.PropTypes.array
-  ]),
-  theme: React.PropTypes.oneOf(['semantic']),
-  valueLink: React.PropTypes.shape({
-    value: React.PropTypes.string,
-    requestChange: React.PropTypes.func
-  })
+  children: t.ReactNode,
+  theme: t.maybe(t.enums.of(['semantic'])),
+  valueLink: t.maybe(t.struct({
+    value: t.maybe(t.String),
+    requestChange: t.Function
+  }))
 };
-
+@props(PropTypes, { strict: false })
```

Comment:
we should add something like "you can pass any other prop and react-select it will be passed to react-select as-is"?